curve_fitting.ipynb

```diff
@@ -155,32 +155,20 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #10.    yfit = sin(xfit, bfit, afit,cfit)
</pre><p>You're calling <code>sin()</code>,not your new <code>cos()</code> .  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/UMD-ENMA165/4-21-22-Class/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>